ralf/plugins/memcached_user_plugin.py

```diff
@@ -0,0 +1,87 @@
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2015  Metaswitch Networks Ltd
```

Comment:
You've checked in the corresponding .pyc files - worth deeting and .gitignore'ing them.